BREAKING_CHANGES.md

```diff
@@ -41,6 +41,31 @@ should be re-checked for ID naming accuracy.
 - `ButtonGroup` now requires it's children to contain a `value` prop which is returned when selected
 - removed `min-width` CSS per design intent
 
+## @zendeskgarden/react-chrome
+
+- No longer packages a `styles.css` dist; CSS is self-contained
+- All irrelevant state and layout styling props (focused, hovered, etc.) have been removed
+- Prop renames:
+  - `Header`
+    - `standalone` -> `isStandalone`
+  - `HeaderItem`
+    - `isRound` -> `isRound`
+    - `logo` -> `hasLogo`
+  - `HeaderItemText`
+    - `clipped` -> `isClipped`
+  - `Nav`
+    - `expanded` -> `isExpanded`
+    - `dark` -> `isDark`
+    - `light` -> `isLight`
```

Comment:
The [readableColor](https://polished.js.org/docs/#readablecolor) utility was exactly what I was looking for! I will update the BREAKING_CHANGES in the CSS-in-JS PR since that will include a new `backgroundColor` (or similar) prop.